packages/calcite-components/src/components/dropdown-item/dropdown-item.tsx

```diff
@@ -125,7 +125,7 @@ export class DropdownItem implements InteractiveComponent, LoadableComponent {
    *
    * @internal
    */
-  @Prop() scale: Scale = "m";
+  @Prop({ reflect: true }) scale: Scale = "m";
```

Comment:
Per our conversation, let's discuss reflection of internal properties as a pattern, as it simplifies styling without significant trade-offs.

Reply:
Reflecting @internal props for styling purposes will help us  improves the maintainability and readability of styles. Though it's internal since they aren't document for public use it is safe to reflect and take advantage of the attribute selector in CSS instead of creating dynamic classes.

Considering this PR as example, reflecting prop helped us avoid cumbersome styling code and there are other instance in the repo where internal props are reflected. We can use this as a pattern if the reflection helps improving the code without affecting UI/UX.

Reply:
Yeah its a trade off discussion...

Is showing an undocumented attribute terrible? No.
Is not showing an undocumented attribute better? Probably.

So its really about if its worth the work to use an internal shadow DOM class vs relying on an attribute.

Reply:
> So its really about if its worth the work to use an internal shadow DOM class vs relying on an attribute.

Agreed. From what I've seen, the extra work of adding root container classes to mimic host and reflected internal attributes (e.g., `scale`-related styling) can feel like overkill. Since we usually avoid reflecting props on the host element, leveraging the reflection of internal properties seems simpler and cleaner in this case.